### ISSUE TITLE
Feature: manipulate snapshot creation logic

### ISSUE THREAD

**Issue by Josipmrden** [memgraph/memgraph#2220]
- ability to disable and enable creation of snapshots at runtime
- ability to change the amount of seconds when snapshots are triggered at runtime
- ability to specify a cron job when the snapshot will be triggered

**Comment by andrejtonev** [comment]
@Josipmrden what exactly do we need to do to enable cron job snapshots?
Wouldn't it be sufficient to just `echo "CREATE SNAPSHOT;" | mgconsole`?

**Comment by Josipmrden** [comment]
Yes but at some point if you start Memgraph with periodical snapshotting, you can't really change that. If there is a big load at some point, you can't disable snapshotting at that time and re-enable it when the big load has passed. (since memory will be increased during the big load and not freed since snapshotting is also a transaction)

**Comment by Josipmrden** [comment]
@andrejtonev  do we close this one?
